spec/lib/secure_headers/headers/policy_management_spec.rb

```diff
@@ -18,7 +18,7 @@ module SecureHeaders
         # (pulled from README)
         config = {
           # "meta" values. these will shape the header, but the values are not included in the header.
-          report_only:  true,     # default: false
+          report_only:  false,
```

Comment:
I'm now validating `report_only` inside the actual config object. So, we can't go setting `report_only: true` unless we are going to be validating it with a `ContentSecurityPolicyReportOnlyConfig`.